conf/modules.config

```diff
@@ -129,7 +129,7 @@ process {
                 "-enzyme no_enzyme",
                 "-subset_max_train ${params.subset_max_train}",
                 "-doc ${params.description_correct_features} ",
-                (params.fdr_level != 'psm-level-fdrs') ? "-" + params.fdr_level : ""
+                (params.fdr_level != 'psm_level_fdrs') ? "-" + params.fdr_level : ""
```

Comment:
check that if this parameter is given that the right annotation is passed to `OPENMS_PERCOLATORADAPTER`, which is still `psm-level-fdrs` as far as I know

Reply:
Then it is fine 👍